core/connector/src/test/java/org/exoplatform/ecm/connector/dlp/TestFileDlpConnector.java

```diff
@@ -160,6 +165,60 @@ public void testProcessItemWhenNotIndexed() throws Exception {
     fileDlpConnectorSpy.processItem(uuid);
     
     // Then
-    Mockito.verify(fileDlpConnectorSpy,Mockito.times(0)).treatItem(uuid);
+    Mockito.verify(fileDlpConnectorSpy,Mockito.times(0)).treatItem(Mockito.eq(uuid), Mockito.any());
+  }
+  
+  @Test
+  public void testGetDetectedKeywords() throws Exception {
+    // Given
+    InitParams initParams = new InitParams();
+    ValueParam dlpKeywordsParam = new ValueParam();
+    dlpKeywordsParam.setName("dlp.keywords");
+    dlpKeywordsParam.setValue("keyword1,keyword2");
+    initParams.addParameter(dlpKeywordsParam);
+    PropertiesParam constructorParams = new PropertiesParam();
+    constructorParams.setName("constructor.params");
+    constructorParams.setProperty("enable", "true");
+    constructorParams.setProperty("displayName", "file");
+    constructorParams.setProperty("type", "file");
+    initParams.addParameter(constructorParams);
+    fileDlpConnector = new FileDlpConnector(initParams, fileSearchServiceConnector, repositoryService, indexingService, queueDlpService);
+    Method getDetectedKeywords = fileDlpConnector.getClass().getDeclaredMethod("getDetectedKeywords", Collection.class, String.class);
+    getDetectedKeywords.setAccessible(true);
+
+    // When
+    String dlpKeywords = "one two three";
+    ArrayList<SearchResult> searchResults = new ArrayList<>();
+    searchResults.add(new SearchResult("url", "title", "excerpt", "detail", "imageUrl", 5, 4));
+
+    ArrayMap<String, List<String>> excerpts = new ArrayMap<>();
```

Comment:
you should use java.util.Hashmap 
```suggestion
    Map<String, List<String>> excerpts = new HashMap<>();
```
Dont forget to remove the import